miniupnpd/Makefile.bsd

```diff
@@ -153,7 +153,7 @@ install:	miniupnpd genuuid
 	$(INSTALL) -d $(DESTDIR)$(INSTALLETCDIR)
 	$(INSTALL) -b miniupnpd.conf $(DESTDIR)$(INSTALLETCDIR)
 	$(INSTALL) -d $(DESTDIR)$(INSTALLMANDIR)
-	$(INSTALL) -m 644 $(SRCDIR)/miniupnpd.8 $(DESTDIR)$(INSTALLMANDIR)/man8/miniupnpd.8
+	$(INSTALL) -m 644 $(SRCDIR)/miniupnpd.8 $(DESTDIR)$(INSTALLMANDIR)/miniupnpd.8
```

Comment:
what is the point of this change ?

Reply:
to fix the directory creation, the line
```
$(INSTALL) -d $(DESTDIR)$(INSTALLMANDIR)
```
should be changed to
```
$(INSTALL) -d $(DESTDIR)$(INSTALLMANDIR)/man8
```
see https://github.com/freebsd/freebsd-ports/blob/main/net/miniupnpd/files/patch-Makefile.bsd